core/test/integration/api/api_notifications_spec.js

```diff
@@ -1,57 +0,0 @@
-var should = require('should'),
-    _ = require('lodash'),
-    uuid = require('uuid'),
-    ObjectId = require('bson-objectid'),
-    testUtils = require('../../utils'),
-    models = require('../../../server/models'),
-    NotificationsAPI = require('../../../server/api/v0.1/notifications');
-
-describe('Notifications API', function () {
-    // Keep the DB clean
-    before(testUtils.teardown);
-    after(testUtils.teardown);
-    before(testUtils.setup('settings', 'users:roles', 'perms:setting', 'perms:notification', 'perms:init'));
-
-    beforeEach(function () {
-        return models.Settings.edit({key: 'notifications', value: '[]'}, testUtils.context.internal);
-    });
-
-    should.exist(NotificationsAPI);
-
-    it('destroy all', function (done) {
```

Comment:
@kirrg001 this one is not covered by routing tests as `destoyAll` method is not exposed over http. Could be covered once we create a unit test suite for notifications controller (didn't go into that rabbit hole for now). lmk what if we should note this in some issue or do something else about it?